tesseract-core/src/lib.rs

```diff
@@ -140,6 +178,8 @@ impl Schema {
         query: &Query,
         ) -> Result<(QueryIr, Vec<String>), Error>
     {
+        // TODO check that cuts have members
```

Comment:
Saw you left this `TODO` here. Sounds like check you added in line 147 of `tesseract-clickhouse/src/sql/primary_agg.rs` so thought maybe you meant to remove this line.

Reply:
I actually meant that I should check that cuts have members at the beginning of `sql_query`, (or maybe on cut parsing?), to make clear that blank members will trigger default hierarchy behavior in sql generation. I'll improve this comment.